include/ignition/rendering/Scene.hh

```diff
@@ -881,6 +881,35 @@ namespace ignition
       public: virtual InertiaVisualPtr CreateInertiaVisual(
                   unsigned int _id, const std::string &_name) = 0;
 
+      /// \brief Create new Joint visual. A unique ID and name will
```

Comment:
```suggestion
      /// \brief Create new joint visual. A unique ID and name will
```
here and below

Reply:
[`4884de6`](https://github.com/ignitionrobotics/ign-rendering/pull/366/commits/4884de6cc4c0b798d9214101f53b726fa73503fe)